packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Live/LiveButton/elements.ts

```diff
@@ -0,0 +1,55 @@
+import styled, { css } from 'styled-components';
+import RecordIcon from 'react-icons/lib/md/fiber-manual-record';
+
+const styles = css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  outline: none;
+  padding: 0.5rem;
+  background-color: #fd2439b8;
+  width: 100%;
+  color: white;
+  border-radius: 4px;
+  font-weight: 800;
+  border: 2px solid #fd2439b8;
+`;
+
+interface ButtonProps extends HTMLButtonElement {
+  disable?: boolean;
```

Comment:
why is this disable and not disabled?

Reply:
I think that's what we had on the button props previously. Feel free to change.